src/plugins/integrations/AWSIntegration.ts

```diff
@@ -139,7 +142,6 @@ class AWSIntegration implements Integration {
               tags: {
                 [SpanTags.SPAN_TYPE]: SpanTypes.AWS_SNS,
                 [AwsSDKTags.REQUEST_NAME]: operationName,
-                [AwsSNSTags.TOPIC_NAME]: topicName,
                 [SpanTags.OPERATION_TYPE]: operationType ? operationType : 'READ',
```

Comment:
we might use empty operation type instead of `READ` if the operation is not known

Reply:
Done